src/components/search/concept-modal/tabs/cdes/cde-item.js

```diff
@@ -19,7 +23,57 @@ const Section = ({ title, children }) => (
     </Space>
 )
 
-export const CdeItem = ({ cde, cdeRelatedConcepts, highlight }) => {
+const RelatedStudiesList = ({relatedStudySource}) => {
+    const { analyticsEvents } = useAnalytics()
+    // const studyLinkClicked = () => {
+    //   analyticsEvents.studyLinkClicked(study.c_id)
+    // }
+    const [showMore, setShowMore] = useState(false)
+
+    if (!relatedStudySource) return (
+        <Text style={{ fontSize: 13, color: "rgba(0, 0, 0, 0.45)" }}>We couldn&apos;t load any related studies.</Text>
+    )
+
+    // Sort the related studies.
+    const relatedStudies = relatedStudySource.sort((a, b) => { return b.c_id < a.c_id});
```

Comment:
This is a bit nitpicky, but typically "source" variables are already sorted in the desired order. Ideally, we'd be sorting them in the fetcher method, CdeItem would also be alright.

Reply:
I'm going to leave this as it is for now.